pkg/rancher-desktop/typings/electron-ipc.d.ts

```diff
@@ -77,6 +77,8 @@ export interface IpcMainEvents {
 
   /** Only for the preferences window */
   'preferences/load': () => void;
+
+  'help/preferences/open-url': () => void;
```

Comment:
Would `preferences/help` be more consistent with the existing `preferences` prefix, or do we anticipate to have many `help` events?

Reply:
We will implement the other events coming from Main window.